### DISCUSSION TITLE
Can smart contract development secure me a job as a new-grad (fresher)?

### DISCUSSION THREAD

**Discussion by sriabhinay556** [Cyfrin/foundry-full-course-cu#2499] [General]
Hello, I’m Abi, a 23-year-old CS grad student with a focus on full-stack web development. I did not have any industry experience tho. I recently started watching Cyfrin and completed the beginner-level courses.

To be honest, I’m not very confident in securing a job in this field, so I've begun researching more through Reddit discussions and popular YouTube video comments. I've noticed that breaking into smart contract development without prior experience is challenging. 

As an international student in the U.S., this time is crucial for me. While I'm excited about this technology, I can’t afford to spend time on something that doesn't offer a meaningful opportunity for employment.

I have a few questions:

1. Is it a good idea to start learning smart contract development as a beginner in the current market conditions? If yes, how?
2. Is the world genuinely adopting blockchain technology? If not now, do you see it becoming relevant soon?
3. How long might it take for this technology to become mainstream and create a significant number of jobs?

Thank you for your help!

**Comment by EngrPips** [comment]
Starting anything is difficult, and standing the chance to achieve anything is even more difficult. I believe that if you are extremely good, you will always stand a good chance wherever you are. It isn't news that blockchain technology is an interesting technology with a great use case.

**Reply by kobbyeugene79** [reply]
100%

**Comment by kobbyeugene79** [comment]
First of all, you have asked very relevant questions, the blockchain technology is difficult to understand if you do not have technical knowledge of the web in general. I have 5 7 years of experience in the web2 space but when I came to the web3 space, I had difficulty grasping certain things. However, with consistency and continuous learning, I know I will become the developer and security researcher I want to be.

1. Yes, it is a good idea to start learning but if you don't have any knowledge in the web2 web development space, I would recommend you learn some basic HTML CSS and JS and get a framework (I personally like VueJs, but you can also use reactJs) to make your work easier and better. You can then continue to learn smart contracts on Cyfrin Updraft as it is the best you can ever find on the internet + it is FREE...
2. Honestly, the world is adopting but not as fast as we want it. With you can others like you, we can increase the adaptation speed.
3. No one in this world can determine that, but as I said, with the help of people like you who want to learn more about web3 technology, we all can educate people around us to get to know blockchain technology and web3 space.

The most important thing here is to learn and educate the people around us. Keep on learning

**Reply by suddenly1990** [reply]
I'm sorry to bother you. I am a Chinese individual currently working in China, where I have been involved in iOS development. Unfortunately, I was laid off from my previous company, which has been quite challenging for me.
At this moment, I am dedicated to learning blockchain technology and actively seeking opportunities in the blockchain field. I've found that the language barrier is not a significant issue for me, as much of the logic appears to be similar to what I've experienced in mobile development.
I would greatly appreciate any insights you might have about the job market for Web3. Thank you for your time and assistance!

**Reply by EngrPips** [reply]
I must confess that I don't have much experience with the Web3 job market but let's look forward to the response of people who do.

**Reply by kobbyeugene79** [reply]
I'm really sorry to hear that, buddy. Don't worry, things will get better for you. Just trust the process and work towards it.

To be honest, most of us here are beginners in the web3 space, including myself. As @EngrPips mentioned, I don't have much experience with the web3 job market. However, building more projects and expanding your portfolio can help you land a good job. If you have strong projects to showcase, you can apply to numerous companies until you get accepted (this approach helped my friend secure his first remote software engineer job in Web2).

Please note that this is not a guarantee, as I am also a beginner like you:
1. Focus on building high-quality projects
2. Keep applying for jobs until you land one
3. Never stop learning
4. Consistency is key
